test/integration/user-management-api/service_plan_features_test.rb

```diff
@@ -75,20 +64,19 @@ def setup
   pending_test 'enable existing feature'
 
   test 'disable feature' do
```

Comment:
I can't spot the difference with `enable new feature` test. This also makes me wonder about naming of `disable non-existing feature` test. I remember discussing this with @josemigallas , maybe he had a PR about this as well?

Reply:
This file is such a mess 😅 I've updated it following API documentation. I think it makes much more sense now